e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/GlobalSearchAO.java

```diff
@@ -143,8 +151,11 @@ public SelenideElement context() {
                 entry(TITLE, context().find(byClassName("review__title"))),
                 entry(DESCRIPTION, context().find(byClassName("review__description"))),
                 entry(HIGHLIGHTS, context().find(byClassName("review__highlights"))),
-                entry(PREVIEW, context().find(byClassName("review__content-preview")))
+                entry(PREVIEW, context().find(byClassName("review__content-preview"))),
+                entry(INFO, context().find(byClassName("review__info"))),
+                entry(INFO_TAB, context().find(byClassName("review__run-table")))
         );
+        private static Condition completed = Condition.or("finished", LogAO.Status.SUCCESS.reached, LogAO.Status.STOPPED.reached, LogAO.Status.FAILURE.reached);
```

Comment:
Line too long. Could you break the value up, please?

Reply:
Done